src/icons/ArrowRight/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ArrowRight = ({ colour }) => (
+  <svg
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+    x="0px"
+    y="0px"
+    width="17"
```

Comment:
do we have to set a width and height or can we make it fill 100% of the parent element? And assume the parent element is set to be 17x13?

Reply:
yeah i'll make it 100% x/y